Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Basic docker image for Monocle
+# Usage:
+#   docker build -t Monocle
+#   docker run -d --name Monocle -P Monocle
+
+FROM python:3.6
```

Comment:
there are a lot of people who know what docker is and may get it started, but acually have no idea of what they are doing ;)